requirements_test.txt

```diff
@@ -5,5 +5,8 @@ coverage~=5.5
 pre-commit~=2.13
 pytest-cov~=2.11
 tbump~=6.3.2
+# Type packages for mypy
 types-typed-ast; implementation_name=="cpython" and python_version<"3.8"
 types-pkg_resources==0.1.2
+types-setuptools==57.4.4
+types-typed-ast==1.5.0
```

Comment:
This is a duplicate. See line **9**:
```
types-typed-ast; implementation_name=="cpython" and python_version<"3.8"
```

If you get type errors in Python >= 3.8, there is probably a `sys.version_info` check missing somewhere.

Reply:
Sorry this is a stupid mistake.. For some reason this wasn't installed locally..

Reply:
I just saw there is in, and I'm working on a fix in https://github.com/PyCQA/astroid/pull/1296.

I'll remove the line and see if this still throws errors after fixing #1296.